website/integrations/services/openproject/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: OpenProject
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is OpenProject
+
+> OpenProject is a open source project management software.
+>
+> -- https://www.openproject.org/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `openproject.company` is the FQDN of the Service install. (Remove this for SaaS)
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+Create a [OAuth2/OpenID provider](https://goauthentik.io/docs/providers/oauth2) with the following parameters:
+
+-   Client Type: `Confidential`
+-   Scopes: `openid`, `email` and `profile`
+-   Signing Key: Select any available key
+-   Redirect URIs: `https://openproject.company/auth/authentik/callback`
+
+Note the `Client ID` and `Client Secret` values.
+Create an [application](https://goauthentik.io/docs/applications), using the provider you've created above and set a slug.
+In this example and the following configuration files, the slug `openproject` is used.
+
+### Add family name
+OpenProject uses `First name` and `Last name` but Authentik does only provide a name by default (e.g. name="foo bar" instead first_name="foo", last_name="bar"), you can modify the `authentik default OAuth Mapping: OpenID 'profile'` to provide first and last name.
+To do that, you need to:
+-   log in as `admin`
+-   open the `admin interface`
+-   navigate to `Customisation` -> `Property Mappings`
+-   uncheck `Hide managed mappings`
+-   edit the `authentik default OAuth Mapping: OpenID 'profile'` mapping
+- add the following lines:
+   ```
+    "family_name": request.user.name.rsplit(" ", 1)[-1],
+    "given_name": request.user.name.rsplit(" ", 1)[0],
+   ```
+
+Now, the fields for first and last name will get properly set in OpenProject.
```

Comment:
In another life, I setup an OpenProject instance, and we would just shove the whoole name in first name, and leave last name empty. It did work a treat, but we can leave the example like this.